conda/cli/main_remove.py

```diff
@@ -219,7 +206,7 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
                         default="no",
                         dry_run=False,
                     )
-                rm_rf(prefix, clean_empty_parents=True)
```

Comment:
This line was added 5 years ago in [this PR](https://github.com/conda/conda/pull/8134/files#diff-54eebab3b20edeedd664816e6d8ae990d901f7b1eb2c4c41f876ba3394bc7189R57), for reference.